PAWS/PAWSAppDelegate.swift

```diff
@@ -24,7 +24,20 @@ class PAWSAppDelegate: CardinalKitAppDelegate {
             if HKHealthStore.isHealthDataAvailable() {
                 HealthKit {
                     CollectSample(
-                        HKQuantityType(.stepCount),
+                        HKQuantityType(.heartRate),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
```

Comment:
I think it might even make sense to get all the HealthKit data in the background as we will most likely want to upload them even if the user did not open the app frequently:
```suggestion
                        deliverySetting: .background(.afterAuthorizationAndApplicationWillLaunch)
```